admin-manual/installation/installation.rst

```diff
@@ -294,9 +294,84 @@ If you have trouble with the gearman command try this as an alternative:
 Install from source
 -------------------
 
-Installing from source has been tested using ansible scripts. Ansible installations have been tested for new installations but are not fully tested for upgrades.
+Installing from source has been tested using ansible scripts.
+Ansible installations have been tested for new installations but are not fully tested for upgrades.
+
+The following instructions detail how to install and run Archivematica from source on a virtual machine.
+
+1. Install VirtualBox, Vagrant, and Ansible:
+
+   .. code:: bash
+
+      sudo apt-get install virtualbox vagrant
+      sudo pip install -U ansible
+
+   Vagrant must be at least version 1.5. Check your version with:
+
+   .. code:: bash
+
+      vagrant --version
+
+   If it is not up to date, you can download the newest version from the `Vagrant website <https://www.vagrantup.com/downloads.html>`_ .
+
+#. Checkout the deployment repo:
+
+   .. code:: bash
+
+      git clone https://github.com/artefactual/deploy-pub.git
+
+#. Download the Ansible roles:
+
+   .. code:: bash
+
+      cd deploy-pub/playbooks/archivematica
+      ansible-galaxy install -f -p roles/ -r requirements.yml
+
+#. Set it to install Archivematica 1.6.  Modify the file `singlenode.yml` to say:
+
+   ..code:: yaml
+
+       - include_vars: "vars-singlenode-1.6.yml"
+
+#. Set it to install in production instead of development mode. Modify the file `vars-singlenode.yml` and **remove** the following line:
+
+   ..code:: ruby
```

Comment:
Possibly! I was only testing the formatting locally, I can't actually confirm if the production version has it. I'd rather not need these instructions if deploy-pub is updated. :)